### PR TITLE
Add turn on mobile data docs

### PR DESCRIPTION
**Pull request by jorunfa**

### CODE REVIEW

File: docs/integrate-android-sdk/setup.md [hunk 1]
```diff
@@ -135,6 +135,18 @@ This is needed for the SDK to work as expected on Android Pie and newer.
 
 If you have your own `networkSecurityConfig` make sure the entries from `network_security_config_connect_id` are included.
 
+## (Optional) Enable prompt for automatic sign in
+
+To allow the SDK to prompt users to turn on mobile data, to allow them to instantly sign in, add the following `meta-data` element to your `AndroidManifest.xml` file:
```
Comment (froeseth) [inline docs/integrate-android-sdk/setup.md]: Consider simplifying the sentence. Perhaps something like this:
"To allow the SDK to prompt users to turn on mobile data so they can instantly sign in, add the following..."

Comment (jorunfa) [reply docs/integrate-android-sdk/setup.md]: Done.

File: docs/integrate-android-sdk/setup.md [hunk 2]
```diff
@@ -135,6 +135,18 @@ This is needed for the SDK to work as expected on Android Pie and newer.
 
 If you have your own `networkSecurityConfig` make sure the entries from `network_security_config_connect_id` are included.
 
+## (Optional) Enable prompt for automatic sign in
```
Comment (ulfcarlin) [inline docs/integrate-android-sdk/setup.md]: Typo: Use noun ("sign-in"), not verb ("sign in")

Comment (jorunfa) [reply docs/integrate-android-sdk/setup.md]: Fixed.

**Pull request merged**